operator/config/crd/bases/machinelearning.seldon.io_seldondeployments.yaml

```diff
@@ -8105,10 +8105,10 @@ spec:
                         type: object
                       endpoint:
                         properties:
-                          grpc_port:
+                          grpcPort:
```

Comment:
Is this change of name not a breaking change?

Reply:
Oh - this should have been in https://github.com/SeldonIO/seldon-core/pull/2574
Looks like regeneration of the CRD was not done.